storage_service/storage_service/settings/components/s3.py

```diff
@@ -4,10 +4,13 @@
 """
 from os import environ
 
+from django.core.exceptions import ImproperlyConfigured
+
 # Read and connect timeouts for S3. Ideally these will match the
 # defaults recommended by your S3 implementation.
 S3_TIMEOUTS = 900
 try:
     S3_TIMEOUTS = int(environ.get("SS_S3_TIMEOUTS", S3_TIMEOUTS))
 except ValueError:
-    pass
+    err_msg = "S3 timeout value configured incorrectly in the environment please check the 'S3_TIMEOUTS' variable"
```

Comment:
Feel free to ignore but adding a hyphen or other separator between "environment" and "please" here might help with the readability of the error message